config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryanalyticshublistings.bigqueryanalyticshub.cnrm.cloud.google.com.yaml

```diff
@@ -6,20 +6,16 @@ metadata:
   creationTimestamp: null
   labels:
     cnrm.cloud.google.com/managed-by-kcc: "true"
-    cnrm.cloud.google.com/stability-level: alpha
     cnrm.cloud.google.com/system: "true"
-    cnrm.cloud.google.com/tf2crd: "true"
   name: bigqueryanalyticshublistings.bigqueryanalyticshub.cnrm.cloud.google.com
 spec:
   group: bigqueryanalyticshub.cnrm.cloud.google.com
   names:
     categories:
     - gcp
     kind: BigQueryAnalyticsHubListing
+    listKind: BigQueryAnalyticsHubListingList
     plural: bigqueryanalyticshublistings
-    shortNames:
-    - gcpbigqueryanalyticshublisting
-    - gcpbigqueryanalyticshublistings
```

Comment:
Can you add the annotations so we don't lose these?

Reply:
Yes and I figured we can have the tooling add it to 🙏🏼  -- https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2887